zuul.d/jobs.yaml

```
@@ -986,83 +1109,125 @@
       - huaweicloud_credentials
 
 - job:
-    name: manageiq-providers-openstack-test-devstack-master
+    name: manageiq-providers-openstack-test-devstack
     parent: init-test
     description: |
       This job run tests of manageiq-providers-openstack aganist Devstack of master.
```

```suggestion
      This job run tests of manageiq-providers-openstack aganist master OpenStack.
```
Same issues exist in following

---

done